app/src/main/java/com/github/libretube/ui/extensions/Activity.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.libretube.ui.extensions
+
+import android.app.Activity
+import androidx.core.view.WindowCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.WindowInsetsControllerCompat
+
+fun Activity.hideSystemBars() {
+    WindowCompat.getInsetsController(window, window.decorView).apply {
+        systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
+        hide(WindowInsetsCompat.Type.systemBars() or WindowInsetsCompat.Type.navigationBars())
```

Comment:
The navigation bars shouldn't be made visible here as that's just a bad practice for UX. We also had some issues open about that previously.